src/lib/events.js

```diff
@@ -47,6 +48,7 @@ export const ERROR_CODE = {
     TOKEN_NOT_VALID: 'error_token_function_not_valid',
     UNSUPPORTED_FILE_TYPE: 'error_unsupported_file_type',
     VIEWER_LOAD_TIMEOUT: 'error_viewer_load_timeout',
+    ZERO_BYTE_FILE: 'error_zero_byte_file',
```

Comment:
Same question as above. Is this a conversion issue or a file issue?

Reply:
This is also an issue with the original file. An example of this case would be creating an empty file with an extension like `empty.pptx`. If the file size is 0 bytes, an attempt to convert it to a different type won't take place.